packages/fx-core/src/component/generator/constant.ts

```diff
@@ -13,3 +13,4 @@ export const templateBetaVersion = "templates-0.0.0-beta";
 export const sampleRepoName = "TeamsFx-Samples";
 export const errorSource = "GEN";
 export const componentName = "generator";
+export const commonTemplateName = "common";
```

Comment:
Personal suggestion: This var can be renamed to like "generalTemplateGroupName".